src/factory.rs

```diff
@@ -319,6 +367,34 @@ impl Factory {
         }))
     }
 
+    /// Create a `Sampler` with default properties.
+    ///
+    /// The default sampler has `Clamp` as its horizontal and vertical
+    /// wrapping mode and `Scale` as its filtering method.
+    pub fn default_sampler(&self) -> Sampler {
+        Sampler(self.default_sampler.clone())
+    }
+ 
+    /// Create new `Sampler`.
+    pub fn sampler(
+        &mut self,
+        filter_method: FilterMethod,
+        horizontal_wrap_mode: WrapMode,
+        vertical_wrap_mode: WrapMode,
+    ) -> Sampler {
+        use gfx::texture::Lod;
+        let info = gfx::texture::SamplerInfo {
```

Comment:
The `Lod` values were copied from [`gfx::texture::SamplerInfo::new()`](https://docs.rs/gfx_core/0.7.1/src/gfx_core/texture.rs.html#455-468).